Jellyfin.Plugin.Anime/Providers/AniList/ApiModel.cs

```diff
@@ -1,6 +1,12 @@
 ﻿using System;
```

Comment:
Most of this file is serialization setup for the JSON parser, attributes come from the [AniList API Docs](https://anilist.gitbook.io/anilist-apiv2-docs/) and mostly the [API v2 GraphQL Docs](https://anilist.github.io/ApiV2-GraphQL-Docs/). There's also methods added to convert values from AniList's representation to Jellyfin's.

The idea is you use Media/MediaSearchResult internally (to the plugin) and convert it to the relevant Jellyfin object type through ToMovie and ToSeries, that way both providers can share most of the code.